### PR TITLE
chore: increase font size in experimental features

### PR DESCRIPTION
**Pull request by ErikSin**
Increasing font size of the new experimental features by request of programs team!

### CODE REVIEW

**Review by achou11 — APPROVED** [review]
Maybe try rerunning tests but otherwise lgtm

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "mapeo-mobile",
-  "version": "5.4.0",
```
Comment (achou11) [inline package-lock.json:3]: what's going on here?

Comment (achou11) [reply package-lock.json:3]: should be 5.3.2 in this case, but curious why there's a diff here 🤔

Comment (achou11) [reply package-lock.json:3]: ah i see this was probably accidentallly changed in https://github.com/digidem/mapeo-mobile/commit/5d3c965ac7cae21b6b9e3049f9edec6606e9a160

**Comment by ErikSin** [comment]
> Maybe try rerunning tests but otherwise lgtm

yeah, something keeps breaking with our CI, its seems to not be working the first time around

**Pull request merged**